src/rt/sched/behaviour.h

```diff
@@ -41,18 +41,18 @@ namespace verona::rt
       return rerun;
     }
 
-    template<typename Be, typename T>
```

Comment:
Given the `invoke` code uses `Be`, could we use `Be` here instead of `T`?  I think this is a hold over from when we supported more elaborate construction.

Reply:
Ah, I was basing it off the `prepare_to_schedule`/`schedule`/`schedule_lambda` code, all of which uses `T`.

Should I rename those to `Be` aswell? (either here or in a follow-up PR)

Reply:
I think that would be good.  Thanks